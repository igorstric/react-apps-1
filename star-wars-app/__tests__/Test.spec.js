// import React from 'react'
// import ReactDOM from 'react-dom'
// import Test from '../src/Test'
// import { shallow } from 'enzyme'
// import toJson from 'enzyme-to-json'
// import Link from '../src/Link';

// describe('<Test />', () => {

//     it('should have ul with 3 children', () => {
//         const wrapper = shallow(<Test />);
//         expect(wrapper.find('ul.t').children().length).toBe(3);
//     });

//     it ('matches the snapshot', () => {
//         const tree = shallow(<Test />);
//         expect(toJson(tree)).toMatchSnapshot();
//     });
// });

// describe('<Link />', () => {

//     it('should link component accepts address prop', () => {
//         const wrapper = shallow(<Link address="www.google.com" />);
//         expect(wrapper.instance().props.address).toBe('www.google.com');
//     });

//     it('should a tag node renders href corectly', () => {
//         const wrapper = shallow(<Link address="www.google.com" />);
//         expect(wrapper.props().href).toBe('www.google.com');
//     });

// });