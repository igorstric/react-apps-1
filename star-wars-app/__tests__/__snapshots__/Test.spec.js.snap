// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Test /> matches the snapshot 1`] = `
<div
  className="Test"
>
  <header>
    <h1>
      Welcome to react
    </h1>
    <p
      className="intro"
    >
      To get testing
    </p>
    <ul
      className="t"
    >
      <li>
        Test1
      </li>
      <li>
        Test2
      </li>
      <li>
        Test3
      </li>
    </ul>
  </header>
</div>
`;
